config/locales/en.yml

```diff
@@ -90,88 +90,90 @@ en:
     layout_super_navigation_header:
       logo_link_title: Go to the GOV.UK homepage
       logo_text: GOV.UK
+      navigation_links:
+      - label: Topics
+        href: "/browse"
+        description: Find help, guidance, and services
+        show: true
```

Comment:
What's this for?

Reply:
It was for me forcing the menu to be open - and now it should be removed

Good 🕵️ skills

Reply:
Removed in https://github.com/alphagov/govuk_publishing_components/pull/2223/commits/f0823c5edbc272c5e218b21134f0ba698af56a3e.